dev/io.openliberty.jakarta.validation.v31_fat/test-applications/val31/src/val31/web/Validation31TestServlet.java

```diff
@@ -80,6 +99,13 @@ public void nestedRecordsTest() throws Exception {
     public void convertGroupsRecordsTest() throws Exception {
         // TODO See section 5.4 of the Bean Validation specification for groups
         // and section 5.4.5 for group conversion
+
+        Registration reg = new Registration("x1asas", true);
```

Comment:
We should be confirming that the groups work, so we should have `isRegistered` be false, confirm that there are no violations when validating `reg`, and then confirm that there is a violation when validating `cmp2`, since cmp2 changes the group to validate on `reg` to `RegistrationChecks`

Reply:
updated